src/pages/creators/Creators.tsx

```diff
@@ -0,0 +1,77 @@
+import { COKO_TEAM_INFO, MEMBER_DETAILS } from '@/features/intro/constants';
+import IntroHeader from '@/features/intro/ui/IntroHeader';
+import Select from '@/features/intro/ui/Select';
+import TutorialPromptModal from '@/features/intro/ui/TutorialPromptModal';
+import useModal from '@/hooks/useModal';
+import * as S from '@/pages/creators/styles';
+import { getImageUrl } from '@/utils/getImageUrl';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+export default function Creators() {
+  const [member, setMember] = useState('홍대경');
+
+  const { isShow, openModal, closeModal, Modal } = useModal();
+
+  const handleSetMember: (value: string) => void = value => {
+    if (!COKO_TEAM_INFO.some(team => team.label === value)) {
+      setMember(value);
+    }
+  };
```

Comment:
```suggestion
  const handleSetMember = (value: string) => {
    if (!COKO_TEAM_INFO.some(team => team.label === value)) {
      setMember(value);
    }
  };
```
함수 자체 타입 정의는 안해도 될 것 같습니다 :)